session/redis/redis_session_service.go

```diff
@@ -519,6 +518,13 @@ func (s *Service) getEventsList(
 		if err != nil {
 			return nil, fmt.Errorf("process event cmd failed: %w", err)
 		}
+
+		// reverse events to get chronological order (oldest first)
+		if len(events) > 1 {
```

Comment:
// reverse events to get chronological order (oldest first)
if len(events) > 1 {
    for i, j := 0, len(events)-1; i < j; i, j = i+1, j-1 { // [<- only line selection]
        events[i], events[j] = events[j], events[i]
    }
}